tests/test_acceptance.py

```diff
@@ -1,30 +1,113 @@
+"""
+Execute a scenario in dry-run mode and compare the results folder with an
+expected output folder, if any difference is found in one of the files,
+the test fails. Note that the empty lines and commented lines
+(the ones that start with #) are not taken into account when computing the
+difference.
+Expected output can be None, in this case the output won't be compared and basic tests will be performed instead
+
+# How to add a new test:
+
+- Create the desired scenario
+- Create a new folder under tests/expected_outputs/test_acceptance/ named after the name
+of this test and fill it with the expected output of the scenario
+(you can copy the output of a valid dry run)
+- Add the path to the scenario and to this expected output folder to SLURM_TEST_SCENARIOS below
+"""
+
 import argparse
 from pathlib import Path
+from typing import Optional
 
 import pytest
 from cloudai.__main__ import handle_dry_run_and_run
 
 SLURM_TEST_SCENARIOS = [
```

Comment:
Let's also split this into two to simplify parametrisation and highlight the difference between the tests.